htdocs/user/param_ihm.php

```diff
@@ -215,6 +215,17 @@
 	$tmparray['ticket/list.php?mainmenu=ticket&leftmenu='] = 'Tickets';
 }
 
+// Hook for insertion new items in the List of possible landing pages
+$reshook = $hookmanager->executeHooks('addToLandingPageList', $tmparray, $object);
+if ($reshook < 0) {
+	setEventMessages($hookmanager->error, $hookmanager->errors, 'errors');
+} elseif ($reshook === 0) {
```

Comment:
You must make a test if 0 or > 0
If > 0, $tmparray=$hookmanager->resArray;
So it replace completely the default tmparray.
If == 0, $tmparray=array_merge($tmparray, $hookmanager->resArray);
So if 0, list is just completed, if >0 it is fully replaced.
Can you enhance this way ?

Reply:
I made the correction. 
Is adding a hook considered as a new feature? (wish I could add it on v16 and v17)